ts/defs.bzl

```diff
@@ -267,19 +236,18 @@ def ts_project(
 
         supports_workers: Whether the worker protocol is enabled.
             To disable worker mode for a particular target set `supports_workers` to `False`.
-            Worker mode can be controlled as well via `--strategy` and `mnemonic` and  using .bazelrc.
 
-            Put this in your .bazelrc to disable it globally: `build --strategy=TsProject=sandboxed`
+            Worker mode can be disabled workspace wide by using the `--@aspect_rules_ts//ts:supports_workers` flag. 
+            To disable worker mode globally, insert `build --@aspect_rules_ts//ts:supports_workers=false` into the .bazelrc.
+
+            Alternatively, worker mode can be controlled via `--strategy`.
+            To disable worker mode globally via `--strategy` insert `build --strategy=TsProject=sandboxed` into the .bazelrc.
```

Comment:
wait, why would we document two different ways to do the same thing? Isn't one of them better?

Reply:
I can't really say that each one has its downsides and upsides and are inherently different even though you get the same result when executing locally. however, you may still want to disable workers when running locally and let the dynamic execution strategy override it when requested. https://bazel.build/remote/dynamic